### ISSUE TITLE
RubberBelts Bob Compatibility Issue

### ISSUE THREAD

**Issue by VasVadum** [ReikaKalseki/Reika_FactorioMods_Issues#226]
Bob's mod has a basic belt, white colored belt made with 1 iron and 1 gear. Its one grade lower than the yellow standard belt.

When I tried this out with it, I didn't get a rubber belt, it stayed iron and gear.

When I check the mod in the mod list, it doesn't specifically have a mod compatibility part in it, I think this forces the game to load your mod after that mod.

![image](https://user-images.githubusercontent.com/1490797/68931207-b3e58700-0755-11ea-834d-5c5bd8f0b4b9.png)
I believe this would make your mod load after his;
![image](https://user-images.githubusercontent.com/1490797/68931254-c8c21a80-0755-11ea-9d8f-f6eac460076e.png)
That is, if thats the problem at least.

**Comment by ReikaKalseki** [comment]
No, the issue is simply that I did not implement a system to change the recipes of other belts. What is the basic belt's internal name?

**Comment by VasVadum** [comment]
I sent this post to Bob so he could try and update you on everything about his mod but he appears too busy. So I'll just give the name of the basic belt and let him add anything extra here if he ever comes around to it.

His belts in order of speed are;
`basic-transport-belt`
`transport-belt`
`fast-transport-belt`
`express-transport-belt`
`turbo-transport-belt`
`ultimate-transport-belt`

**Comment by ReikaKalseki** [comment]
Oh, they changed again.
